CAM Files/2 Layer.cam

```diff
@@ -166,12 +167,12 @@
                         "file_function": "Paste",
                         "layer_type": "bottom"
                     },
-                    "filename_format": "%PREFIX/solderpaste_bottom.gbr",
+                    "filename_format": "%PREFIX/%N.GPB",
```

Comment:
I think this should be `GBP` not `GPB`. All the other files start with `GB` or `GT`.